packages/ember-data/tests/integration/adapter/find-all-test.js

```diff
@@ -10,6 +10,9 @@ module("integration/adapter/find_all - Finding All Records of a Type", {
       firstName: DS.attr('string'),
       lastName: DS.attr('string')
     });
+    //TODO mock for now
+    Person.typeKey = 'person';
+    Person.modelName = 'person';
```

Comment:
Since #3033 is merged and `references` is rebased onto `master`, this can be removed.